media-video/ffmpeg/ffmpeg-3.2.2.ebuild

```diff
@@ -238,6 +238,7 @@ RDEPEND="
 		>=x11-libs/libXv-1.0.10[${MULTILIB_USEDEP}]
 	)
 	xcb? ( >=x11-libs/libxcb-1.4[${MULTILIB_USEDEP}] )
+	zeromq? ( >=net-libs/zeromq-4.1.6 )
```

Comment:
Then you need to disable it for non native ABIs like it is done for frei0r and netcdf. It is preferable to add multilib support to zmq though.